rate_counter.go

```diff
@@ -84,133 +84,131 @@ func NewStandardRateCounter(numSamples int64, samplePeriodMs int64, clock clock.
 	return rc
 }
 
-func (this *StandardRateCounter) Clear() {
-	this.lock.Lock()
-	defer this.lock.Unlock()
+func (sRC *StandardRateCounter) Clear() {
```

Comment:
nit: don't capitalize initialisms (`src`)

Would be better to do this style change as its own PR.